### ISSUE TITLE
Fresh install in Ubuntu 20.04 won't start

### ISSUE THREAD

**Issue by pimterry** [OpenShot/openshot-qt#3890]
**Describe the bug:**

I've just installed openshot from the Ubuntu 20.04 repo, in a fairly fresh OS install, and it crashes straight away. It never successfully starts.

**Steps to reproduce the behavior:**

Unclear - but on my machine, just run `openshot-qt`. Plausibly, do that on any fresh Ubuntu 20.04 system. 

**Expected behavior:**

Openshot should start. I suspect there's a bad dependency or something, but it's not clear what and this may be a widespread issue. At the very least fixing this and documenting it here might help others who hit the same error.

**System Details:**

- OpenShot Version [e.g. 2.4.3]: 2.4.3
- Operating System / Distro: Regolith (closely based on Ubuntu 20.04, OpenShot installed from Ubuntu repos)

- `dpkg -S openshot`:
  ```
  Package: openshot
  Status: install ok installed
  Priority: optional
  Section: oldlibs
  Installed-Size: 13
  Maintainer: Ubuntu Developers <ubuntu-devel-discuss@lists.ubuntu.com>
  Architecture: all
  Source: openshot-qt
  Version: 2.4.3+dfsg1-1
  Depends: openshot-qt
  Description: create and edit videos and movies (transitional package)
   This is a transitional package. It can safely be removed.
  Original-Maintainer: Debian Multimedia Maintainers <debian-multimedia@lists.debian.org>
  Homepage: https://www.openshot.org/
  ```

This machine was freshly installed a couple of weeks ago. As far as I'm aware, there's nothing particularly unusual in its setup. I'm not a python developer and AFAIK the Python configuration is still the default.

**Log Files:**

`libopenshot.log` is empty, `openshot-qt.log` contains the same exception output as the command (below).

**Exception / Stacktrace:**

```
➜ openshot-qt
Compiled translation resources missing!
Loading translations from: /usr/lib/python3/dist-packages/openshot_qt/language
Loaded modules from installed directory: /usr/lib/python3/dist-packages/openshot_qt
      launch:INFO ------------------------------------------------
      launch:INFO    OpenShot (version 2.4.3)
      launch:INFO ------------------------------------------------
         app:INFO openshot-qt version: 2.4.3
         app:INFO libopenshot version: 0.2.2
         app:INFO platform: Linux-5.4.0-53-generic-x86_64-with-glibc2.29
         app:INFO processor: x86_64
         app:INFO machine: x86_64
         app:INFO python version: 3.8.5
         app:INFO qt5 version: 5.15.2
         app:INFO pyqt5 version: 5.15.2
     metrics:ERROR Error determining OS version in metrics.py
    language:INFO Qt Detected Languages: ['en-GB', 'en']
    language:INFO LANG Environment Variable: en_GB.UTF-8
    language:INFO LOCALE Environment Variable: 
    language:INFO OpenShot Preference Language: Default
project_data:INFO Setting default profile to HD 720p 30 fps
ZmqLogger::Connection - Error binding to tcp://*:5556. Switching to an available port.
ZmqLogger::Connection - Error binding to tcp://*:5556. Switching to an available port.
         app:INFO Setting font to Cantarell
logger_libopenshot:INFO Connecting to libopenshot with debug port: 5556
         app:INFO Setting custom dark theme
  exceptions:ERROR Unhandled Exception
Traceback (most recent call last):
  File "/usr/bin/openshot-qt", line 11, in <module>
    load_entry_point('openshot-qt==2.4.3', 'gui_scripts', 'openshot-qt')()
  File "/usr/lib/python3/dist-packages/openshot_qt/launch.py", line 98, in main
    app = OpenShotApp(sys.argv)
  File "/usr/lib/python3/dist-packages/openshot_qt/classes/app.py", line 157, in __init__
    from windows.main_window import MainWindow
  File "/usr/lib/python3/dist-packages/openshot_qt/windows/main_window.py", line 43, in <module>
    from windows.views.timeline_webview import TimelineWebView
  File "/usr/lib/python3/dist-packages/openshot_qt/windows/views/timeline_webview.py", line 40, in <module>
    from PyQt5.QtWebKitWidgets import QWebView
ValueError: PyCapsule_GetPointer called with incorrect name
```

**Screenshots:** (Optional)

N/A

**Comment by USATechDude** [comment]
Version 2.4.3 is no longer supported. I don't know if this is related but I just found out on my Ubuntu 20.04.1 LTS install that Python2 is deprecated and somewhat tricky to reinstall.  If OpenShot v2.4.3 uses Python2, that could be your issue.  I suggest purging v2.4.3 and using the PPA of OpenShot v2.5.1.  The instructions to load the repository are on the [download webpage](https://www.openshot.org/ppa/).

**Comment by Innitinnuit** [comment]
Try sudo apt upgrade python3-openshot.

**Comment by pimterry** [comment]
Uninstalled everything, I think, tried installing from the stable PPA: now crashes with https://github.com/OpenShot/openshot-qt/issues/3615 instead.

`python3-openshot` is up to date (`0.2.5+dfsg2+876+202003032113+daily~ubuntu20.04.1`)

Added daily PPA, upgraded to the openshot version there, now throws:

```
RROR exceptions: Unhandled Exception
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/openshot_qt/windows/views/webview.py", line 165, in <module>
    from .webview_backend.webengine import TimelineWebEngineView as WebViewClass
  File "/usr/lib/python3/dist-packages/openshot_qt/windows/views/webview_backend/webengine.py", line 38, in <module>
    from PyQt5.QtWebEngineWidgets import QWebEngineView, QWebEnginePage
ModuleNotFoundError: No module named 'PyQt5.QtWebEngineWidgets'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/usr/bin/openshot-qt", line 11, in <module>
    load_entry_point('openshot-qt==2.5.1.dev2', 'gui_scripts', 'openshot-qt')()
  File "/usr/lib/python3/dist-packages/openshot_qt/launch.py", line 140, in main
    app = OpenShotApp(argv)
  File "/usr/lib/python3/dist-packages/openshot_qt/classes/app.py", line 256, in __init__
    from windows.main_window import MainWindow
  File "/usr/lib/python3/dist-packages/openshot_qt/windows/main_window.py", line 82, in <module>
    from windows.views.webview import TimelineWebView
  File "/usr/lib/python3/dist-packages/openshot_qt/windows/views/webview.py", line 175, in <module>
    raise RuntimeError(
RuntimeError: Need PyQt5.QtWebEngine (or PyQt5.QtWebView on Win32)
```

I've checked my `python3-pyqt5` is already the latest available version (`5.14.1+dfsg-3build1`). Manually installed `python3-pyqt5.qtwebengine`, still shows the same error.

I've switched to the AppImage build directly from https://www.openshot.org/download/. That seems to work nicely, so I'm going to ignore this for now, but it's probably worth somebody investigating further, if the PPA/Ubuntu distributions are expected to work in these cases.
